src/main/java/com/mateacademy/spring_mvc/service/UserService.java

```diff
@@ -0,0 +1,16 @@
+package com.mateacademy.spring_mvc.service;
+
+import com.mateacademy.spring_mvc.entity.User;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface UserService {
+    List<User> getAllUsers();
+
+    void saveUser(User user);
+
+    Optional getUserById(Long id);
```

Comment:
if you return Optional then return parameterized one `Optional<User> `
but here in service return just User without optional